R/zzz.R

```diff
@@ -0,0 +1,3 @@
+.onLoad <- function(libname, pkgname) {
+  try(proj_set(".", quiet = TRUE), silent = TRUE)
+}
```

Comment:
Why is this needed now, but not previously? I don't really understand how project activation is supposed to work.

Reply:
Oh, there is no new need! In most usage scenarios, this doesn't matter. You do notice it when developing usethis itself. This activates a project after `load_all()`, which is just nice.

I would say project activation is supposed to work "on first demand".

Another consequence of auto-activate, more relevant to other people: All `use_*()` functions that operate on a project eventually make a `proj_get()` and that will activate a project. But if the first usage is of a helper like `write_union()`, that won't activate a project, but it *will* honour an already-activated project. This does not change what is written or where, but makes the path reported in the message nicer (i.e. relative to the project instead of absolute).

Reply:
If auto-activation seems like a bad idea, I could also deal with the usethis development hangnail via something in the test helper file.